frontend/device/sdl/device.lua

```diff
@@ -70,6 +70,7 @@ local Device = Generic:new{
     startTextInput = SDL.startTextInput,
     stopTextInput = SDL.stopTextInput,
     canOpenLink = getLinkOpener,
+    canControlCharge = yes, -- development of batteryCare
```

Comment:
please make it  a property of the Emulator, not a property of a generic SDL device.

Reply:
Isn't this redundant with hasBattery btw?

Reply:
@Frenzie: It is not redundant, as hasBattery tells us there is a battery and canControlCharge tells us we can turn charging on/off programmatically.

Reply:
Then it definitely doesn't belong here.